CHANGELOG.md

```diff
@@ -7,6 +7,17 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## Unreleased
 
+### Added
+
+- iCal export final schedule oncall [#1501](https://github.com/grafana/oncall/issues/1501)
+- Schedules: Allow web-edited overrides for Terraform-managed rotations [828](https://github.com/grafana/oncall/issues/828)
```

Comment:
FYI, these 2 above were already released and available (so I would remove them from here)